### ISSUE TITLE
decommission Dolores bot

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#52]
_Add/remove items, as applicable._

- [x] Notify any users of the system, via banner across the site, email, Slack, etc. Ideally, help them find alternatives, make sure they're off in time, etc.
- [x] Export any relevant [records](https://handbook.tts.gsa.gov/records-management/) and store in Google Drive - [file](https://drive.google.com/file/d/1dX_XH4DlHVDgls8ImC6onrJJh-v4pKHV/view?usp=sharing)
- [x] Clean up infrastructure
  - [x] [Ask cloud.gov](https://cloud.gov/contact/) for the organization to be deleted.
  - [x] Delete any associated AWS resources, and decommission any AWS accounts. Ask in [#infrastructure](https://gsa-tts.slack.com/messages/infrastructure) for help.
  - [x] If the site was operated on Federalist, contact [federalist-support@gsa.gov](mailto:federalist-support@gsa.gov) to deprovision the [custom domain](https://cloud.gov/docs/management/custom-domains/) for the site
- [x] [Archive the GitHub repositor(ies)](https://help.github.com/articles/archiving-repositories/). Ask in [#admins-github](https://gsa-tts.slack.com/messages/admins-github) if you need help.
- [x] Domain changes
  - [x] Clean up any DNS entries that are no longer needed, from [the DNS repository](https://github.com/18F/dns) or elsewhere - https://github.com/18F/dns/pull/483
- [x] Notify other parts of GSA
  - [x] Notify your ISSM
  - [x] Notify the Privacy Office ([gsa.privacyact@gsa.gov](mailto:gsa.privacyact@gsa.gov)) to take down the [privacy impact assessment (PIA)](https://www.gsa.gov/reference/gsa-privacy-program/privacy-impact-assessments-pia) and/or privacy threshold assessment (PTA)
- [x] Clean up external services
  - [x] Remove (sub)accounts
    - [x] New Relic - [#admins-newrelic](https://gsa-tts.slack.com/messages/admins-newrelic)
  - [x] Remove any external integrations
    - [x] [Slack](https://handbook.tts.gsa.gov/slack/#integrations)
  - [x] Archive the Slack channel
  - [x] Archive the issue tracker (Trello, etc.)

More info:

- [Notes](https://docs.google.com/document/d/1aOkolVTn3QhkLdBSS7dy2hEXoz6UwGx9PAg-fpXE2Pw/edit#bookmark=id.hh9eqy35i3d0)
- [TTS orgs in cloud.gov](https://docs.google.com/spreadsheets/d/1Bdzl9n2E1MXWV4elXvZ-nYuZmmEj4PEU-u5aZlNGZF4/edit#gid=1352278899)
- [Decommissioning checklist](https://before-you-ship.18f.gov/infrastructure/decommissioning/)

**Comment by mgwalker** [comment]
Not to argue the intent here, but a minor clarification that [Charlie](https://github.com/18F/18f-bot) isn't "abandoned" in the sense that it's actively maintained (probably gets some attention once a month or so). However, it doesn't have a well-defined owner and is floating around in space, so it's kind of abandoned in that sense.

**Comment by afeld** [comment]
Narrowed to just be about Dolores.

**Comment by afeld** [comment]
- [ ] [Get added to the org in cloud.gov](https://gsa-tts.slack.com/archives/C09CR1Q9Z/p1604342016133800)

**Comment by afeld** [comment]
Just waiting on cloud.gov org deletion.

**Comment by ccostino** [comment]
Ah, memories, one of the first things I worked on when I joined 18F!

*Pours one out for the Dolores Landingham bot and its occasional tendency to "landingham" alums...* 😆

The cloud.gov `gsa-18f-dolores` org and all of its contents has been deleted:

<img width="1264" alt="Screen Shot 2020-11-02 at 11 34 20 PM" src="https://user-images.githubusercontent.com/121565/97950159-00114900-1d64-11eb-974a-0edf161a4c2e.png">

**Comment by afeld** [comment]
Yeah, I was responsible for shutting down [C2](https://github.com/18f/c2) recently, which was _my_ first project. A couple of TTS geezers, we are!

Thanks for the help!
